frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/common/ErrorMessages.java

```diff
@@ -23,6 +23,7 @@
     public static final String COMPONENT_DUPLICATE ="Component could not be added, since a component with the same name already exists.";
     public static final String RELEASE_NOT_ADDED = "Release could not be added.";
     public static final String RELEASE_DUPLICATE ="Release could not be added, since a release with the same name and version already exists.";
+    public static final String DUPLICATE_ATTACHMENT ="Multiple Attachments with same name cannot be present in attachment list.";
```

Comment:
I am a fan of proper formatting as you might have noticed :-) Maybe add a space after the equal sign (I know, the others did not do it consistently, but we can do it better :D ) and write "attachments" with a small A.

Reply:
Done also updated the error message as we will be checking file name along with sha1.